.github/workflows/sync_crowdin.yml

```diff
@@ -26,10 +28,10 @@ jobs:
           download_translations: true
           localization_branch_name: feat/translations
           create_pull_request: true
-          pull_request_title: "chore(i18n): Sync translations"
-          pull_request_body: "Sync translations from [Crowdin/ReVanced](https://crowdin.com/project/revanced)"
+          pull_request_title: "chore: Sync translations"
+          pull_request_body: "Sync translations from [crowdin.com/project/revanced](https://crowdin.com/project/revanced)"
           pull_request_base_branch_name: "dev"
-          commit_message: "chore(i18n): Sync translations"
+          commit_message: "chore: Sync translations"
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
As far as I know the checkout step will keep the credentials in cache so that other steps can use it. Is this necessary? @Ushie